internal/sponge/item/item_test.go

```diff
@@ -159,3 +160,41 @@ func TestGetAll(t *testing.T) {
 		}
 	}
 }
+
+// TestInferId tests the inference of an item_id from type and source id.
+func TestItemizeData(t *testing.T) {
+	tests.ResetLog()
+	defer tests.DisplayLog()
+
+	// Test the inference of item_id where source_id is present.
+	d, err := itemfix.GetData("data.json")
+	if err != nil {
+		t.Fatalf("\t%s\tShould be able to load item data.json fixture: %v", tests.Failed, err)
+	}
+	it := item.Item{
```

Comment:
Odd.  I am and golint did not catch this when running manually from the command line. Will continue to look into why.